imageGalleryPage/image-gallery.html

```diff
@@ -7,7 +7,7 @@
     <link rel="stylesheet" href="./gallery-styles.css">
     <link rel="preconnect" href="https://fonts.gstatic.com">
     <link href="https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@300&family=Shrikhand&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css" integrity="sha512-c42qTSw/wPZ3/5LBzD+Bw5f7bSF2oxou6wEb+I/lqeaKV5FDIfMvvRp772y4jcJLKuGUOpbJMdg/BTl50fJYAw==" crossorigin="anonymous" />
```

Comment:
i'd move this up before your gallery-styles.css in case you want to adjust styles from animate.css. browsers are fast and should just work despite the location of the link tag. but it's good to not risk it. we download animate first and then gallery-styles.